sycl/test/regression/bit_cast_unit32_to_half2.cpp

```diff
@@ -0,0 +1,10 @@
+// RUN: %clangxx -fsycl -fpreview-breaking-changes %s -o %t.out
+#include <sycl/sycl.hpp>
+
+SYCL_EXTERNAL uint32_t test(uint32_t a) {
+   sycl::half2 ah=sycl::bit_cast<sycl::half2, uint32_t>(a);
+   return sycl::bit_cast<uint32_t, sycl::half2>(ah);
+}                                                                    
+int main() {
+   return 0;
+}
```

Comment:
```suggestion
// RUN: %clangxx -fsycl -fpreview-breaking-changes -c %s -o %t.out
#include <sycl/sycl.hpp>

SYCL_EXTERNAL uint32_t test(uint32_t a) {
   sycl::half2 ah=sycl::bit_cast<sycl::half2, uint32_t>(a);
   return sycl::bit_cast<uint32_t, sycl::half2>(ah);
}                                                                    
```

Reply:
But this gives undefined reference to main error.